src/main.ts

```diff
@@ -163,18 +180,7 @@ function serve(config: webpack.Configuration, args: any): Promise<void> {
 		})
 	);
 
-	const outputDir = (config.output && config.output.path) || process.cwd();
-	if (args.mode === 'dist' && Array.isArray(args.compression)) {
-		const useBrotli = args.compression.includes('brotli');
-		app.use(
-			expressStaticGzip(outputDir, {
-				enableBrotli: useBrotli,
-				orderPreference: useBrotli ? ['br'] : undefined
-			})
-		);
-	} else {
-		app.use(express.static(outputDir));
-	}
+	serveStatic(app, outputDir, args.mode, args.compression);
```

Comment:
This is the second time `serveStatic` is called within `serve`, without any `return` or conditional in between. Is this deliberate?